include/AudioSdl.h

```diff
@@ -66,8 +66,10 @@ class AudioSdl : public AudioDevice
 		void saveSettings() override;
 
 	private:
-		QLineEdit * m_device;
+		QComboBox* m_playbackDeviceComboBox;
```

Comment:
Maybe add in a `nullptr` here too?

Reply:
Good catch! I remember that both combo boxes were defined as `nullptr`. However, in #5990 it was corrected via commit https://github.com/LMMS/lmms/pull/5990/commits/c1e5de5b3b9ee2137a98b20d05f110ce7d078b28 which added the results of a code review. However, I could not include that commit it because it also includes changes with regards to the recording feature.

Reply:
Fixed with commit https://github.com/LMMS/lmms/pull/7421/commits/2f1736a9f567ff0d53c33baaaac89970c7f9d025 which also fetches some other changes made to `AudioSdl` in "mixed" commits.